selfdrive/controls/lib/longitudinal_mpc_lib/long_mpc.py

```diff
@@ -136,7 +142,7 @@ def gen_long_mpc_solver():
            x_ego,
            v_ego,
            a_ego,
-           20*(a_ego - prev_a),
```

Comment:
I previously saw some differences when moving the cost out from here, I'm not sure if they're significant though.